src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -24,8 +24,6 @@
     <Compile Remove="commands\utility\FormatAndOutput\OutGridView\OutWindowProxy.cs" />
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Verify approved aliases list.All approved aliases have the correct 'ReadOnly' option
```powershell
Expected $null or empty, but got @(@{InputObject=lp -> Out-Printer; SideIndicator==>}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 577
577:         $result | Should -BeNullOrEmpty
```